### ISSUE TITLE
Strange result

### ISSUE THREAD

**Issue by simmed00** [SimVascular/svOneDSolver#49]
I tested the simple artery model, using outlet pressure = 0, the pressure difference (pressure loss) calculated from the segment is quite different from the theoretical value using equation like:

Ploss = Q * 8 * mu * l / pi / (D^4). 

Has this been tested before?

**Comment by mrp089** [comment]
Hi @simmed00,

Yes, the svOneDSolver has been compared to analytical solutions and numerical 3D and 0D simulations. You can have a look at the references at the bottom of our documentation page: https://simvascular.github.io/docs1DSimulation.html

Without knowing your exact setup and results, you could have a look at the following common error sources:
- D in your formula should be the radius (not the diameter)
- the sign of the prescribed inflow should be positive
- inflow and boundary condition defined at the correct points

Hope this helps! For a more detailed error analysis, I would need your input file.

Martin

**Comment by simmed00** [comment]
Thank you very much for the prompt reply and correction. 
My setup is like below. It is a straight artery (R=0.2cm, L=10cm), I fix the outlet pressure to be 0 (ref pressure = 0 too). I then ramp up the inlet flow rate to 10cm^3 (I guess the unit is cm^3?) in 1 second and then keep it constant for another second (to get steady state result). I get the result of P=0.186 at t=2 (I guess the unit is 0.1Pa, if pressure is expressed in cm.g.sec) 

Using the classical equation, Ploss = 636Pa, it is quite different from the oned solver result. 


MODEL straight

NODE 0 0.0 0.0 0.0
NODE 1 0.0 0.0 10.0

SEGMENT ARTERY0 0 10.0 50 0 1 0.1257 0.1257 0.0 MAT1 NONE 0.0 0 0 PRESSURE OUTLETDATA

DATATABLE INLETDATA LIST
0.0 0.0
1.0 10.0
2.0 10.0
ENDDATATABLE

DATATABLE OUTLETDATA LIST
0.0 0.0
ENDDATATABLE

MATERIAL MAT1 LINEAR  1.06 0.04 0 1.0 1.0e7

SOLVEROPTIONS 0.001 200 2000 2 INLETDATA FLOW 1.0e-6 1 1 

OUTPUT TEXT

**Comment by wgyang** [comment]
Use the following material option:

"MATERIAL MAT1 OLUFSEN 1.06 0.04 0 2.0 0.0 0.0 1.0e7"

It should produce a pressure drop of 636 Pa.

FYI, for "Olufsen" material,  P=Po+4/3*EHR*(1-sqrt(Ao/A)), EHR=k1*exp(k2*ro)+K3, so k1=0.0, k2=0.0, and k3=1e7 lead to  EHR=1e7.

**Comment by wgyang** [comment]
Here is the input file.

MODEL straight

NODE 0 0.0 0.0 0.0
NODE 1 0.0 0.0 10.0

SEGMENT ARTERY0 0 10.0 50 0 1 0.1257 0.1257 0.0 MAT1 NONE 0.0 0 0 PRESSURE OUTLE
TDATA

DATATABLE INLETDATA LIST
0.0 0.0
1.0 10.0
2.0 10.0
ENDDATATABLE

DATATABLE OUTLETDATA LIST
0.0 0.0
ENDDATATABLE

MATERIAL MAT1 OLUFSEN 1.06 0.04 0 2.0 0.0 0.0 1.0e7

SOLVEROPTIONS 0.001 200 2000 2 INLETDATA FLOW 1.0e-6 1 1

OUTPUT TEXT

**Comment by mrp089** [comment]
Than you for your input @simmed00! After some debugging, we found there is a problem with the current master. Right now, we suggest to checkout commit 9837884. This should reproduce the analytical result in the test case posted by @wgyang.

**Comment by mrp089** [comment]
@simmed00 We fixed the bug in 50edf9c. Please give it another try! 

We also added comparisons to analytical results in test/tube_*.in

Thanks for your help!
